coalib/testing/LocalBearTestHelper.py

```diff
@@ -149,15 +154,15 @@ def verify_local_bear(bear,
     :param invalid_files:    An iterable of files as a string list that must
                              yield results.
     :param filename:         The filename to use for valid and invalid files.
-    :param settings:         A dictionary of keys and values (both string) from
-                             which settings will be created that will be made
-                             available for the tested bear.
     :param force_linebreaks: Whether to append newlines at each line
                              if needed. (Bears expect a \\n for every line)
     :param create_tempfile:  Whether to save lines in tempfile if needed.
     :param timeout:          The total time to run the test for.
     :param tempfile_kwargs:  Kwargs passed to tempfile.mkstemp() if tempfile
                              needs to be created.
+    :param settings:         A dictionary of keys and values (both string) from
+                             which settings will be created that will be made
+                             available for the tested bear.
```

Comment:
The change for this function does belong to a second commit, though I wouldn't change this at all^^